### ISSUE TITLE
"init()" should be called first

### ISSUE THREAD

**Issue by cheetosysst** [naver/egjs-flicking#655]
## Description
I'm using nextjs and egjs-flicking. I tried copy pasting the [example](https://naver.github.io/egjs-flicking/docs/quick-start) to my code. It caused some error: 
```
Unhandled Runtime Error
FlickingError: This module is not attached to the Flicking instance. "init()" should be called first.
```
I not very familiar with nextjs, so I couldn't figure out what went wrong.

## Steps to check or reproduce
```jsx
import Flicking from "@egjs/react-flicking";
import "@egjs/react-flicking/dist/flicking.css";
// Or, if you have to support IE9
import "@egjs/react-flicking/dist/flicking-inline.css";

export default function carousel() {
	return (
		<Flicking
			align="prev"
			circular={true}
			onMoveEnd={(e) => {
				console.log(e);
			}}
		>
			<div className="panel">1</div>
			<div className="panel">2</div>
			<div className="panel">3</div>
		</Flicking>
	);
}
```

**Comment by cheetosysst** [comment]
```
Uncaught (in promise) FlickingError: This module is not attached to the Flicking instance. "init()" should be called first.
    FlickingError flicking.esm.js:809
    getFlickingAttached flicking.esm.js:487
    updatePanelSize flicking.esm.js:5023
    resize flicking.esm.js:8009
    step flicking.esm.js:188
    verb flicking.esm.js:119
    fulfilled flicking.esm.js:74
    promise callback*step flicking.esm.js:89
    __awaiter flicking.esm.js:92
    __awaiter flicking.esm.js:71
    resize flicking.esm.js:7982
    init flicking.esm.js:7526
    step flicking.esm.js:188
    verb flicking.esm.js:119
    __awaiter flicking.esm.js:92
    __awaiter flicking.esm.js:71
    init flicking.esm.js:7496
    Flicking flicking.esm.js:6620
    componentDidMount flicking.esm.js:508
    React 8
    workLoop scheduler.development.js:266
    flushWork scheduler.development.js:239
    performWorkUntilDeadline scheduler.development.js:533
    EventHandlerNonNull* scheduler.development.js:571
    <anonymous> scheduler.development.js:633
    NextJS 4
    <anonymous> index.js:6
    NextJS 4
    React 2
    NextJS 4
    <anonymous> React
    NextJS 4
    <anonymous> index.js:10
    NextJS 4
        js
        factory
        __webpack_require__
        fn
    <anonymous> next-dev.js:2
    NextJS 7
        js
        factory
        __webpack_require__
        __webpack_exec__
        <anonymous>
        webpackJsonpCallback
        <anonymous>
flicking.esm.js:809
```
Detail message

**Comment by WoodNeck** [comment]
Hello @cheetosysst!
I think this is maybe a duplication of #643 if you're using React@18.
I've fixed that issue in the new version (v4.6.0), please try that!

**Comment by cheetosysst** [comment]
Yes, it works after updating to 4.6. Thank you for the reply!
